docs/en/ingest-management/elastic-agent/configuring-kubernetes-metadata.asciidoc

```diff
@@ -0,0 +1,41 @@
+[[configuring-kubernetes-metadata]]
+= Configuring kubernetes metadata enrichment on {agent} managed by {fleet}
+
```

Comment:
Add a single line what kubernetes metadata enrichment is 

Maybe a link here https://www.elastic.co/guide/en/observability/current/monitor-kubernetes.html#beats-metadata